src/components/listNavBar/index.tsx

```diff
@@ -1,14 +1,13 @@
-import React from "react";
 import { useStyles } from "./styles";
-import ArrowCircleRightIcon from "@mui/icons-material/ArrowCircleRight";
-import ArrowCircleLeftIcon from "@mui/icons-material/ArrowCircleLeft";
-import { Button } from "@material-ui/core";
+import { Pagination } from "@mui/material";
 import { globalVariables } from "../../styles/globalVariables";
+import React, { Dispatch, SetStateAction } from "react";
+
 type Props = {
   page: number;
   totalPages: number;
-  setPageNumber: React.Dispatch<React.SetStateAction<number>>;
-  setPageSize: React.Dispatch<React.SetStateAction<number>>;
+  setPageNumber: Dispatch<SetStateAction<number>>;
+  setPageSize: Dispatch<SetStateAction<number>>;
```

Comment:
IF  setPageSize and pageSize are not being used then they can be removed from props